buildconf

```diff
@@ -1,4 +1,28 @@
 #!/bin/sh
+#
+#  +----------------------------------------------------------------------+
+#  | PHP Version 7                                                        |
+#  +----------------------------------------------------------------------+
+#  | Copyright (c) The PHP Group                                          |
+#  +----------------------------------------------------------------------+
+#  | This source file is subject to version 3.01 of the PHP license,      |
+#  | that is bundled with this package in the file LICENSE, and is        |
+#  | available through the world-wide-web at the following url:           |
+#  | http://www.php.net/license/3_01.txt                                  |
+#  | If you did not receive a copy of the PHP license and are unable to   |
+#  | obtain it through the world-wide-web, please send a note to          |
+#  | license@php.net so we can mail you a copy immediately.               |
+#  +----------------------------------------------------------------------+
+#  | Authors:                                                             |
+#  +----------------------------------------------------------------------+
```

Comment:
Even if we can't remove existing ones, let's not add new license headers where there were none ;)

Reply:
To leave out the entire header? This applies for the new files also?

Reply:
Or this applies for only authors section?

Reply:
Leave out the whole header.

It should be added to new C/H files for consistency, but there is no need to stick it everywhere, especially if it wasn't there before. We have a few ten thousand test files without license headers, you know ;)

Reply:
Ok.